docs/authentication/enterprise-connections/saml/azure.mdx

```diff
@@ -17,39 +17,60 @@ description: Learn how to allow users to sign up and sign in to your Clerk app w
     }
   ]}
 >
-  - Use Microsoft Azure Entra ID to enable SSO via SAML for your Clerk app
+  - Create a SAML SSO connection for your customer
+  - Give your customer instructions on how to connect Microsoft Azure Entra ID to your app
+  - Enable and test the SAML connection
 </TutorialHero>
 
 Enabling SAML with Microsoft Azure Entra ID (formerly [Active Directory](https://learn.microsoft.com/en-us/entra/fundamentals/new-name)) allows your users to sign up and sign in to your Clerk application with their Microsoft account.
 
-To make the setup process easier, it's recommended to keep two browser tabs open: one for the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [Microsoft Azure portal](https://portal.azure.com).
+This process requires configuration changes in both the Clerk dashboard, and in your customer's Microsoft Entra ID settings in their Azure account.
 
 <Steps>
-  ## Enable Microsoft Entra ID as a SAML connection in Clerk
+  ## Create a Microsoft Entra ID SAML connection in Clerk
 
   1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
   1. Select **Add connection** and select **For specific domains or organizations**.
   1. Under **SAML**, select **Microsoft Entra ID (Formerly AD)**.
   1. Enter the **Domain**. This is the email domain of the users you want to allow to sign in to your application. Optionally, select an **Organization**.
   1. Enter the **Name**. This will be displayed on the sign-in form.
-  1. Select **Add connection**. You'll be redirected to the connection's configuration page.
-  1. In the **Service Provider Configuration** section, save the **Reply URL (Assertion Consumer Service URL)** and **Identifier (Entity ID)** values somewhere secure. Keep this page open.
+  1. Select **Add connection**. You'll be redirected to the connection's configuration page. Note that the connection is disabled by default.
+  1. In the **Service Provider Configuration** section, note the **Reply URL (Assertion Consumer Service URL)** and **Identifier (Entity ID)**. These will need to be added to your customer's Microsoft Entra ID application.
 
-  ## Create a new enterprise app in Microsoft
+  ## Configure the third-party Identity Provider
```

Comment:
```suggestion
  ## Configure SAML application
```

What if we remove `third-party` here, curious what was your reasoning to include it! I'd say we should mention "SAML application" instead if configuring the identity provider

Identity Provider is the "whole database of identities", and SAML application is the actual federation bucket within their system to connect to the Clerk app